site/en/docs/extensions/mv3/manifest/manifest_version/index.md

```diff
@@ -2,27 +2,23 @@
 layout: "layouts/doc-post.njk"
 title: "Manifest Version"
 date: 2013-05-12
-updated: 2018-04-26
+updated: 2021-04-19
 description: Reference documentation for the manifest_version property of manifest.json.
 ---
 
-One integer specifying the version of the manifest file format your package requires. As of Chrome
-18, developers _should_ specify `2` (without quotes) to use the format as described by this
-document:
+An integer specifying the version of the manifest file format your package requires. For example:
 
 ```json
-"manifest_version": 2
+"manifest_version": 3
 ```
 
-Consider manifest version 1 _deprecated_ as of Chrome 18. Version 2 is not yet _required_, but we
-will, at some point in the not-too-distant future, stop supporting packages using deprecated
-manifest versions. Extensions, applications, and themes that aren't ready to make the jump to the
-new manifest version in Chrome 18 can either explicitly specify version `1`, or leave the key off
-entirely.
+Legal values for this key are:
 
-The changes between version 1 and version 2 of the manifest file format are described in detail in
-[the `manifest_version` documentation.][1]
+* 3: Use the [Manifest V3][mv3] format and associated feature set.
+* 2: Use the [Manifest V2][mv2] format and associated feature set.
 
-<div class="aside aside--caution">Setting <code>manifest_version</code> 2 in Chrome 17 or lower is not recommended. If your extension needs to work in older versions of Chrome, stick with version 1 for the moment. We'll give you ample warning before version 1 stops working.</div>
+The current version is Manifest V3. Manifest V2 is also currently permitted, but will be phased out
+in the near future. Other manifest versions greater than 3 are not planned yet.
```

Comment:
> Other manifest versions greater than 3 are not planned yet.

(Perhaps just me): While we don't have concrete plans or timelines, we do intend to continue iterating on the platform, and I imagine there _will_ be a MV4 (and it will likely be fewer than 8 years away :)).  I wonder if this gives the wrong impression?  Is there phrasing we could use to more accurately convey that, while MV4 is not currently concrete in any way, that's not to say we anticipate MV3 being the final version ever?

Reply:
Done.